app/src/features/observations/BlockObservationPage.tsx

```diff
@@ -140,6 +160,41 @@ const BlockObservationPage = () => {
     return <CircularProgress className="pageProgress" size={40} />;
   }
 
+  const handleSaveAndExit = async (values: ICreateBlockObservationPostRequest) => {
+    if (!formikRef?.current) {
+      return;
+    }
+
+    const postData: any = {
+      block_name: formikRef.current.values.block_name,
+      start_datetime: moment(formikRef.current.values.date + ' ' + formikRef.current.values.start_time).toISOString(),
+      end_datetime: moment(formikRef.current.values.date + ' ' + formikRef.current.values.end_time).toISOString(),
```

Comment:
can we use template strings here to construct this

Reply:
moment(`${formikRef.current.values.date} ${formikRef.current.values.start_time}`).toISOString()